.github/workflows/ci.yml

```diff
@@ -0,0 +1,76 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - 'master'
+
+jobs:
+  test:
+    name: OTP ${{ matrix.otp_version }}
+    runs-on: ubuntu-latest
+    container: erlang:${{ matrix.otp_version }}
+
+    strategy:
+      matrix:
+        include:
+        - otp_version: 23
+          elixir_version: 1.10.3
+          new_rebar: true
+          # hdr_histogram up to 0.4.0 fails to compile with OTP 23
+          env:
+            XPROF_ERL_HIST: true
```

Comment:
Did you mean `false`?

Reply:
thanks for reviewing :) its the NIF hdr_hist that does not compile, so have to use the erlang hist implementation, hence ERL_HIST=true. But it does not work like this because matrix does not seem to support `env:` :(